src/tenant/tests/test_initialization.py

```diff
@@ -42,15 +42,15 @@ def test_superusers_created(self):
         accounts = {usernames[i]: passwords[i] for i in range(len(usernames))}
 
         for username, password in accounts.items():
```

Comment:
I think it would be better to explicitly test each individual user that is created.  There are only two, and it makes it much clearer what is being test.  In fact, they should probably be in their own tests.  test_owener_create and test_admin_created